rust/agents/relayer/src/msg/metadata/mod.rs

```diff
@@ -0,0 +1,8 @@
+mod base;
+mod multisig;
+mod routing;
+
+pub(crate) use base::BaseMetadataBuilder;
+pub(crate) use base::MetadataBuilder;
+pub(crate) use multisig::MultisigIsmMetadataBuilder;
+pub(crate) use routing::RoutingIsmMetadataBuilder;
```

Comment:
Looks like you _might_ just be using these in the other submodules, if that is the case I don't think you need to make them `pub(crate)` because `pub(self)` (the default) actually exposes to submodules. So `multisig` should be able to access `super::base`.